### PR TITLE
[ID] Add `Beatmap ranking procedure/Ranking queue`

### PR DESCRIPTION
**Pull request by hikikuman**
Add Indonesian translation for [Beatmap ranking queue](https://osu.ppy.sh/wiki/en/Beatmap_ranking_procedure/Ranking_queue) and update outdated Indonesian translation for [Beatmap ranking procedure](https://osu.ppy.sh/wiki/en/Beatmap_ranking_procedure).

### CODE REVIEW

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - ranking ETA
+  - antrian peringkat
+  - tahap kualifikasi
+---
+
+# Antrian peringkat beatmap
+
+**Antrian peringkat** mengatur pemindahan [beatmap-beatmap](/wiki/Beatmap) dari [Qualified](/wiki/Beatmap/Category#qualified) ke [Ranked](/wiki/Beatmap/Category#ranked). Setiap hari, 10 beatmap dari masing-masing [mode permainan](/wiki/Game_mode) dipindahkan dari Qualified ke Ranked, jika beatmap-beatmap tersebut sudah berada di kategori Qualified setidaknya selama 7 hari. Waktu di mana beatmap-beatmap tersebut menjadi ranked adalah acak.
+
+## Disqualification and re-qualification
```
Comment (wowcake) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:14]: remove this line

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_ranking_procedure/id.md [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
 
-[Modding](/wiki/Modding) adalah proses dari kegiatan yang bertujuan untuk menerima dan membangun kritik guna meningkatkan kualitas sebuah beatmap. Pengguna biasanya mengirimkan umpan balik mereka pada laman diskusi beatmap yang tersedia atau melakukan diskusi atau daring secara langsung dengan mapper beatmap yang dimaksud.
+[Modding](/wiki/Modding) adalah proses dari kegiatan yang bertujuan untuk menerima dan membangun kritik guna meningkatkan kualitas sebuah beatmap. Pengguna biasanya mengirimkan umpan balik mereka pada laman diskusi beatmap yang tersedia atau melakukan diskusi secara langsung dengan mapper beatmap yang dimaksud.
 
 Para beatmapper baru biasanya membutuhkan modding ekstensif supaya beatmap yang mereka buat memenuhi standar kualitas kategori Ranked dan tidak jarang mereka mendapat kritik seperti lebih baik membuat ulang beatmap mereka dari awal.
 
-Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmapset membutuhkan postingan [hype](/wiki/Glossary#hype) dari lima pengguna berbeda.
+Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmap membutuhkan postingan [hype](/wiki/Glossary#hype) dari lima pengguna berbeda.
```
Comment (DoryanR) [inline wiki/Beatmap_ranking_procedure/id.md:15]:
```suggestion
Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmap membutuhkan postingan [hype](/wiki/Beatmap/Hype) dari lima pengguna berbeda.
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - qualified queue
+  - ranked queue
+  - ranking ETA
+  - antrian peringkat
+  - tahap kualifikasi
+---
+
+# Antrian peringkat beatmap
+
+**Antrian peringkat** mengatur pemindahan [beatmap-beatmap](/wiki/Beatmap) dari [Qualified](/wiki/Beatmap/Category#qualified) ke [Ranked](/wiki/Beatmap/Category#ranked). Setiap hari, 10 beatmap dari masing-masing [mode permainan](/wiki/Game_mode) dipindahkan dari Qualified ke Ranked, jika beatmap-beatmap tersebut sudah berada di kategori Qualified setidaknya selama 7 hari. Waktu di mana beatmap-beatmap tersebut menjadi ranked adalah acak.
```
Comment (mousesalt) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:12]:
```suggestion
**Antrian peringkat** atau *Beatmap ranking queue* mengatur pemindahan [beatmap-beatmap](/wiki/Beatmap) dari keadaan [Qualified](/wiki/Beatmap/Category#qualified) ke [Ranked](/wiki/Beatmap/Category#ranked). Setiap hari, 10 beatmap dari masing-masing [mode permainan](/wiki/Game_mode) dipindahkan dari Qualified ke Ranked, jika beatmap-beatmap tersebut sudah berada di kategori Qualified setidaknya selama 7 hari. Waktu di mana beatmap-beatmap tersebut menjadi ranked adalah acak.
```

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Disqualification and re-qualification
+
+## Diskualifikasi dan mengulang kualifikasi
```
Comment (mousesalt) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:16]:
```suggestion
## Diskualifikasi dan re-kualifikasi
```

Comment (mousesalt) [reply wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:16]: if `Dis...` was used for it, the next should be consistent.

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+Saat sebuah beatmap di[diskualifikasi](/wiki/Beatmap_ranking_procedure#pengulangan-nominasi), waktu yang sudah berjalan saat berada di kategori Qualified akan tetap tersimpan. Jika beatmap tersebut telah dikualifikasi ulang, beatmap tersebut akan masuk ke antrian peringkat dan melanjutkan sisa waktu yang sebelumnya sudah ditempuh. Kemampuan untuk "melewati" waktu yang telah dihabiskan dalam batas antrian pada 6 hari adalah untuk memastikan bahwa beatmap-beatmap tersebut selalu berada di kategori Qualified setidaknya selama satu hari penuh.
```
Comment (mousesalt) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:18]:
```suggestion
Saat sebuah beatmap terkena [diskualifikasi](/wiki/Beatmap_ranking_procedure#pengulangan-nominasi), waktu yang sudah berjalan saat berada di kategori Qualified menuju Ranked akan tetap tersimpan. Jika beatmap tersebut telah dikualifikasi lagi, beatmap tersebut akan masuk ke antrian peringkat dan melanjutkan sisa waktu yang sebelumnya sudah ditempuh. Kemampuan untuk "melewati" waktu yang telah dihabiskan dalam batas antrian pada 6 hari adalah untuk memastikan bahwa beatmap-beatmap tersebut selalu berada di kategori Qualified setidaknya selama satu hari penuh.
```

**Review by mousesalt — COMMENTED** [review]
the commit of Indonesian ranking procedure was merge https://github.com/ppy/osu-wiki/pull/5046

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md (hunk 3 above)
Comment (hikikuman) [reply wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:12]: `*Beatmap ranking queue*` diubah jadi `*Ranking queue*` karena di en.md nya bukan tertulis `*Beatmap ranking queue*`. Kata `keadaan` opsional aja sebenernya, kalo gak ditambahin juga gak apa2. Kalo lebih tepatnya sih harusnya pake `kategori`, karena sumber wikinya itu kan beatmap category.

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md (hunk 1 above)
Comment (hikikuman) [reply wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:14]: my bad ):

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md (hunk 5 above)
Comment (hikikuman) [reply wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:18]: `di kategori Qualified` itu udah paten artinya sedang menuju Ranked, jadi gak perlu pake `menuju Ranked` juga seharusnya udah cukup. Sisanya dah tak apply 👍

**Comment by hikikuman** [comment]
Deleted ranking procedure from this PR.

**Review by mousesalt — APPROVED** [review]

File: wiki/Beatmap_ranking_procedure/Ranking_queue/id.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  - qualified queue
+  - ranked queue
+  - ranking ETA
+  - antrian peringkat
+  - tahap kualifikasi
+---
+
+# Antrian peringkat beatmap
+
+**Antrian peringkat** (*Ranking queue*) mengatur pemindahan [beatmap-beatmap](/wiki/Beatmap) dari [Qualified](/wiki/Beatmap/Category#qualified) ke [Ranked](/wiki/Beatmap/Category#ranked). Setiap hari, 10 beatmap dari masing-masing [mode permainan](/wiki/Game_mode) dipindahkan dari Qualified ke Ranked, jika beatmap-beatmap tersebut sudah berada di kategori Qualified setidaknya selama 7 hari. Waktu di mana beatmap-beatmap tersebut menjadi ranked adalah acak.
```
Comment (TicClick) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:12]: I'm curious why it's "beatmap-beatmap" -- honest mistake or a proper term?

Comment (wowcake) [reply wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:12]: it's a way to represent the plural (s) (to some extent) in indonesian, by doubling the object next to it

beatmaps = beatmap-beatmap
mods= mod-mod
modes = mode-mode

something like that \o/

Comment (TicClick) [reply wiki/Beatmap_ranking_procedure/Ranking_queue/id.md:12]: wow!

**Review by TicClick — APPROVED** [review]

**Pull request merged**